### ISSUE TITLE
docopt

### ISSUE THREAD

**Issue by ghost** [deanishe/alfred-mpd#1]
Hey, Love the plugin!

Unfortunately, it won't work because of this bug:

```
 File "./ampd", line 40, in <module>
    from lib.docopt import docopt
ImportError: No module named lib.docopt
```

I did run `pip install docopt` still didn't do the job!
Any tips? :)

**Comment by ghost** [comment]
Ok, i think you simpy forgot to add the lib's into the workflow. I manually copied them, but now I have another error:

```
03:35:29 workflow.py:2243 ERROR    invalid literal for int() with base 10: 'n'
Traceback (most recent call last):
  File "/Users/macuser/Dropbox/_Documents/__Backups/Alfred.alfredpreferences/workflows/user.workflow.9EC1E456-A85E-4A86-827D-D87ACDC6DF17/lib/workflow/workflow.py", line 2236, in run
    func(self)
  File "./ampd", line 683, in main
    return do_search(opts)
  File "./ampd", line 484, in do_search
    return do_stats(opts)
  File "./ampd", line 623, in do_stats
    playing = mpd.playing()
  File "/Users/macuser/Dropbox/_Documents/__Backups/Alfred.alfredpreferences/workflows/user.workflow.9EC1E456-A85E-4A86-827D-D87ACDC6DF17/lib/mpd.py", line 386, in playing
    return status().playing
  File "/Users/macuser/Dropbox/_Documents/__Backups/Alfred.alfredpreferences/workflows/user.workflow.9EC1E456-A85E-4A86-827D-D87ACDC6DF17/lib/mpd.py", line 362, in status
    return _parse_status(out)
  File "/Users/macuser/Dropbox/_Documents/__Backups/Alfred.alfredpreferences/workflows/user.workflow.9EC1E456-A85E-4A86-827D-D87ACDC6DF17/lib/mpd.py", line 304, in _parse_status
    volume = int(v.rstrip('%'))
ValueError: invalid literal for int() with base 10: 'n'
03:35:29 workflow.py:2246 INFO     For assistance, see: https://github.com/deanishe/alfred-mpd/issues
03:35:29 workflow.py:2267 DEBUG    Workflow finished in 0.062 seconds.
```

:)

**Comment by deanishe** [comment]
>  i think you simpy forgot to add the lib's into the workflow

Bug in the build script.

> another error

What does Alfred's debugger say? Which version of mpc are you using?

**Comment by ghost** [comment]
I just installed the latest mpc from brew. I counld't find a way to find the version. my mpd is on `0.20.9`.

The thing i posted up there is from the Alfred debugger!

**Comment by deanishe** [comment]
Sorry. When you run `mpc status` on the command line, what's the output?

**Comment by ghost** [comment]
```
Harry Gregson-Williams - The King
[playing] #9/15   1:25/5:45 (24%)
volume: n/a   repeat: off   random: off   single: off   consume: off
```

(That's the Track i'm currently playing.)

**Comment by deanishe** [comment]
Thanks.

Should be fixed in [v0.1.2](https://github.com/deanishe/alfred-mpd/releases/tag/v0.1.2).

**Comment by ghost** [comment]
Yay! I works.. and it looks HOT especially in combination with ncmpcpp!

Thank you a lot!
